recipes/ablog/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "ablog" %}
+{% set version = "0.9.2" %}
+{% set sha256 = "4027fbf75826bac4ad413eb5c8ab8e21bfe8e8c2635a07d9326873d9f14a27b7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - ablog = ablog.commands:ablog_main
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - werkzeug
+    - sphinx
+    - alabaster
+    - invoke
+    - python-dateutil
+    - sphinx-automodapi
+
+test:
+  commands:
+    - cd docs
```

Comment:
I think we need to git clone the repo first and and then cd into the doc folder. 

```
- git clone --depth 1 git@github.com:sunpy/ablog.git
- cd ablog/docs
```

Reply:
Not sure if the https link would be better: https://github.com/sunpy/ablog.git

Reply:
See https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#source-files

Reply:
Thanks, that would be better!

So  that should be

```
test:
  source_files:
    - docs
```

Reply:
Of course this wont work! We dont put the docs in the PyPi setup. 

I think we need to do the git clone instead.